parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -1408,9 +1409,15 @@ declareForeigns = do
   declareForeign "IO.setBuffering.impl.v3" set'buffering
     . mkForeignIOF $ uncurry hSetBuffering
 
-  declareForeign "IO.getBytes.impl.v3" boxNatToEFBox .  mkForeignIOF $ \(h,n) -> Bytes.fromArray <$> hGet h n
+  declareForeign "IO.getLine.impl.v3" boxToEFBox . mkForeignIOF
+    $ \h -> pack <$> hGetLine h
```

Comment:
This probably doesn't make much of a difference, but there's an `hGetLine` in `Data.Text.IO`.

Reply:
Yes that seems better and it looks like that uses the encoding set on the Handle: http://hackage.haskell.org/package/text-1.2.4.1/docs/Data-Text-IO.html#g:3